tests/unit/intelligence/test_get_workspace_debug_context_tool.py

```
@@ -0,0 +1,723 @@
+    assert restored.recent_changes == []
+    assert restored.related_tests == []
+    assert restored.inferred_commands == []
+
+
+# ---------------------------------------------------------------------------
+# 9. Pydantic-level schema: LaunchConfig field requirements
+# ---------------------------------------------------------------------------
+
+
+def test_launch_config_requires_name():
+    with pytest.raises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)